app/coffeescripts/views/conferences/ConferenceView.coffee

```diff
@@ -126,3 +128,63 @@ define [
         else
           window.open(data[0].url)
       )
+
+    delete_recording: (e) ->
+      return if !confirm(I18n.t('recordings.confirm.delete', "Are you sure you want to delete this recording?"))
+      e.preventDefault()
+      parent = $(e.currentTarget).parent()
+      params = {recording_id: parent.data("id")}
+      this.toggleActionButton(parent, {state: "processing", action: "delete"})
+      this.toggleRecordingLink(parent, {state: "processing"})
+      $.ajaxJSON(parent.data('url') + "/delete_recording", "POST", params,
+        (data) =>
+          if data.deleted
+            this.removeRecordingRow(parent)
+            return
+          this.ensure_delete_performed({attempt: 1, parent: parent, desired_state: "true"})
```

Comment:
why even pass `desired_state` if it's always true?

Reply:
Right, this comes from a more generic function we used before. The current function "ensure_delete_performed" used to be "ensure_action_performed". The validation was based on the parameter because it was used for different types of actions.

"published", "protected", etc

But now that we are only implementing delete, well it can be removed